src/NuGet.Clients/NuGet.CommandLine/Commands/HelpCommand.cs

```diff
@@ -91,13 +91,16 @@ orderby c.CommandAttribute.CommandName
         // Help command always outputs NuGet version
         protected override bool ShouldOutputNuGetVersion { get { return true; } }
 
-        private void PrintCommand(int maxWidth, CommandAttribute commandAttribute)
+        private void PrintCommand(int maxWidth, CommandAttribute commandAttribute, DeprecatedCommandAttribute deprecatedCommandAttribute = null)
         {
             // Write out the command name left justified with the max command's width's padding
             Console.Write(" {0, -" + maxWidth + "}   ", GetCommandText(commandAttribute));
             // Starting index of the description
             int descriptionPadding = maxWidth + 4;
-            Console.PrintJustified(descriptionPadding, commandAttribute.Description);
+
+            var deprecatedWord = deprecatedCommandAttribute != null ? $"({deprecatedCommandAttribute.DeprecatedWord()}) " : string.Empty;
+
+            Console.PrintJustified(descriptionPadding, deprecatedWord + commandAttribute.Description);
```

Comment:
The two lines above are causing multiple string concatenations. You make make avoid any string concatenation in most cases, and only a sting concat using:

```
var description = deprecatedCommandAttribute != null
    ? deprecatedCommandAttribute.DeprecatedWord() + " " + commandAttribute.Description
    : commandAttribute.Description;
```